src/app-layout/visual-refresh/app-bar.tsx

```diff
@@ -64,6 +64,7 @@ export default function AppBar() {
             className={testutilStyles['navigation-toggle']}
             ref={focusRefsNav.toggle}
             disabled={isAnyPanelOpen}
+            __nativeAttributes={{ 'aria-haspopup': true }}
```

Comment:
Side note: why not expose the `ariaHaspopup` in the public API? If we have a good use case for it -- it is likely that our customers would have it as well.